libs/store/src/lib/reducers/remove.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  AppStateModel,
+  AppPlayerLogId,
+  AppRaceId,
+  AppPlayerId,
+} from '@razor/models';
+import { removePlayerReducerPayload } from '../payloads';
+
+// Basic Remove Operations
+export const removePlayerReducer = (
+  state: AppStateModel,
+  payload: removePlayerReducerPayload,
+): AppStateModel => {
+  const { tournamentId, playerId } = payload;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { [playerId]: removedPlayer, ...newPlayerModel } = state.playersModel;
+  const newState: AppStateModel = {
+    ...state,
+    tournamentsModel: {
+      ...state.tournamentsModel,
+      [tournamentId]: {
+        ...state.tournamentsModel[tournamentId],
+        playerIds: [
```

Comment:
you can simplify this as below:
`state.tournamentsModel[tournamentId].playerIds.filter((id) => id !== playerId)`

Reply:
simplified b6085c6c6aa6ad83c276be895e541d9b685b0414